processors/include/TrackingAnaProcessor.h

```diff
@@ -113,11 +115,24 @@ class TrackingAnaProcessor : public Processor {
         bool doTruth_{false}; //!< description
         int isData_{1}; //! is data
         int debug_{0}; //!< debug level
+        int seed_{0};  //!< seed
         float time_offset_{0}; //! time offset
-
+        
+        //Momentum smearing closure test
         std::shared_ptr<TrackSmearingTool> smearingTool_;
+        std::shared_ptr<TrackSmearingTool> smearingToolRel_;
+        std::string pSmearingFile_{""};
+        TH1D* psmear_h_;
```

Comment:
should we add these as plots in the TrackingHistos instead?

Reply:
I don't think so. They are explicitly helpful only for those interested in studying the difference smearing can cause. Everyone else would just like smearing to be applied (or not).